exercises/zebra-puzzle/example.go

```diff
@@ -158,6 +173,8 @@ func Solve() (waterDrinker string, zebraOwner string) {
 	return
 }
 
+var showHouseFacts bool = false // when true, print all house facts for fun.
```

Comment:
`const` is better. 

Also drop type bool since it's inferred.

Reply:
Ok.